components/builder-web/app/client/builder-api.ts

```diff
@@ -315,7 +319,9 @@ export class BuilderApiClient {
           if (response.ok) {
             resolve(response.json());
           } else {
-            reject(new Error(response.statusText));
+            response.json().then(error => {
```

Comment:
I think this can just be 
`response.json().then(reject);`

(haven't tested it though)

Reply:
Yeah, that's a little cleaner, huh -- I'll try it.